actions/image_partition_action.go

```diff
@@ -45,7 +45,10 @@ configuration (below) and label the filesystem located on this partition.
 
 - fs -- filesystem type used for formatting.
 
-NB: the FAT (vfat) filesystem is named 'fat32' in configuration file.
+'none' fs type should be used for partition without filesystem.
+
+NB: the FAT (vfat) filesystem have an alias for 'fat32' in configuration file
```

Comment:
*has an alias

Also i don't really see how that's relevant for the user; The user can use either fat32 or vfat as both names tend to be used.